doc/source/installation.rst

```diff
@@ -0,0 +1,70 @@
+Installation
+============
+
+Depending on your experience, role, computing environment and expectations optimal hardware utilization, you have several ways to install the dependencies required to run PIConGPU.
```

Comment:
role is equal to what a person is doing in an organization. the next sections clearify that (sysadmin, admin, user in various environments.

this page is a stub and can be improved later on (like all pages I added initially).